backend/app/Http/Controllers/User/ShowEditUserController.php

```diff
@@ -231,14 +232,16 @@ public function __invoke(): View|Factory
              * 市区町村情報セット
              */
 
-            $preBirthMunicipalities = Area::where('prefecture_code', $userLiveArea->prefecture_code->value())->get();
-            $preLiveMunicipalities = Area::where('prefecture_code', $userBirthArea->prefecture_code->value())->get();
-            if (isset($userInfo->u_u_faculty_id)) {
-                $prePreUUMajor = UUMajor::where('id', $userInfo->u_u_faculty_id)->first();
+            $preBirthMunicipalities = Area::where('prefecture_code', $userBirthArea->prefecture_code->value())->get();
+            $preLiveMunicipalities = Area::where('prefecture_code', $userLiveArea->prefecture_code->value())->get();
+            if (isset($userInfo->u_u_major_id)) {
+                $prePreUUMajor = UUMajor::where('id', $userInfo->u_u_major_id)->first();
                 $preUUMajors = UUMajor::where('faculty_id', $prePreUUMajor->faculty_id->value())->get();
             }
         }
-        \Log::debug($preUUMajors);
+
+        \Log::debug($userLiveArea->prefecture_code->value());
```

Comment:
**[phpstan]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Parameter #1 $message of static method Illuminate\Log\Logger::debug() expects array|Illuminate\Contracts\Support\Arrayable|Illuminate\Contracts\Support\Jsonable|Illuminate\Support\Stringable|string, int given.